Makefile

```diff
@@ -12,13 +12,16 @@ pull-kakarot: .gitmodules
 build-kakarot: setup 
 	cd lib/kakarot && make build && make build-sol
 
+build-testing-solidity-contracts:
+	FOUNDRY_PROFILE=testing forge build
```

Comment:
testing profile allows us to specify two different source directories. i am leaving the `out` compilation output as the same, so i wouldnt need to dig too deeply in the assumptions that the `deploy_helpers` code I wrote makes about where to look for deployed contracts